### DISCUSSION TITLE
Kedro 0.18.2 is out!

### DISCUSSION THREAD

**Discussion by merelcht** [kedro-org/kedro#1705] [Announcements]
We're excited to announce that Kedro 0.18.2 has finally been released! :tada:

> **TL;DR** This release is special because we've worked on improving Databricks Repos support and improved the CLI experienced by adding in pretty logging to the CLI with [Rich](https://github.com/Textualize/rich) and reducing the `kedro new` prompts to one question.

What can you expect in this release?

:rocket: Major features:
- Kedro now uses the [Rich](https://github.com/Textualize/rich) library to format terminal logs and tracebacks.
- The file `conf/base/logging.yml` is now optional. See [our documentation](https://kedro.readthedocs.io/en/0.18.2/logging/logging.html) for details.
- Introduced a `kedro.starters` entry point. This enables plugins to create custom starter aliases used by `kedro starter list` and `kedro new`.
- Reduced the `kedro new` prompts to just one question asking for the project name.

:bug: Bug fixes:
- Bumped `pyyaml` upper bound to make Kedro compatible with the [pyodide](https://pyodide.org/en/stable/usage/loading-packages.html#micropip) stack.
- Kedro's framework-side logging configuration no longer performs file-based logging. Hence superfluous `info.log`/`errors.log` files are no longer created in your project root, and running Kedro on read-only file systems such as Databricks Repos is now possible.
- Required `cookiecutter>=2.1.1` to address a [known command injection vulnerability](https://security.snyk.io/vuln/SNYK-PYTHON-COOKIECUTTER-2414281).
- The session store no longer fails if a username cannot be found with `getpass.getuser`.
- Added generic typing for `AbstractDataSet` and `AbstractVersionedDataSet` as well as typing to all datasets.

You can find the full list of changes in [the release notes](https://github.com/kedro-org/kedro/blob/main/RELEASE.md).
[Kedro 0.18.2](https://github.com/kedro-org/kedro/releases/tag/0.18.1) is a non-breaking release in the 0.18.x series, upgrade now with:

How do I get access to the latest release?
Get the latest version of Kedro with ```pip install kedro == 0.18.2``` or install Kedro from Conda with ```conda install -c conda-forge kedro==0.18.2```.

**Comment by mjmare** [comment]
I like rich but can I turn off the new Tracebacks? IMO they are a complete waste of space, sometimes making one scroll for pages and pages to find the actual error.

**Reply by antonymilne** [reply]
Yes! See https://github.com/kedro-org/kedro/issues/1728 for how you can do this now. Also please do add your opinion here so we can work out what better default options might be: https://github.com/kedro-org/kedro/issues/1750.

**Reply by antonymilne** [reply]
Basically you can alter the tracebacks with the following code in your settings.py:
```
# no locals and only 1 extra line context
rich.traceback.install(suppress=[click, str(Path(sys.executable).parent)], extra_lines=1)
```
Or turn off rich tracebacks altogether with:
```
import sys
sys.excepthook = sys.__excepthook__
```

**Reply by mjmare** [reply]
I tried both options.
The first does not work for me: still a mess in the PyCharm terminal (regardless of the "emulate terminal in output console") and the line with the error is not clickable. 
The second works. Thank you!
